pkg/cmd/kubeconfig/kubeconfig.go

```diff
@@ -137,28 +127,36 @@ func (o *options) Run(f util.Factory) error {
 		return err
 	}
 
+	o.RawConfig = rawConfig
+
+	return nil
+}
+
+// Run does the actual work of the command.
```

Comment:
If we do not impl Validate the base Validate is called.
```go
func (o *options) Validate() error {
	if o.RawConfig == nil {
		return new Error("Raw Config is required")
	}

	return nil
}
```

Reply:
✅